template/gemma3-instruct.gotmpl

```diff
@@ -0,0 +1,16 @@
+{{- $skipStartOfTurn := false }}
```

Comment:
is `skipStartOfTurn` needed?

Reply:
yes, otherwise the `<start_of_turn>user` segment gets added both for the assistant and the user.